File: 2018/pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Declaring integer variables
    int height, row, space, dash;

    // Prompt user for height
    do
    {
        height = get_int("Height: ");
    }
    while (height < 0 || height > 23);

    // Print rows
    for (row = 0; row < height; row++)
    {
        // Print spaces for left half pyramid
        for (space = (height - row - 1); space > 0; space--)
        {
            printf(" ");
        }

        // Print dashes for left half pyramid
        for (dash = 0; dash < row + 1; dash++)
        {
            printf("#");
        }

        // Print spaces in middle
        for (int i = 1; i <= 1; i++)
        {
            printf("  ");
        }

        // Print dashes in right half pyramid
        for (dash = 0; dash < row + 1; dash++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: 2018/pset6/sentimental/mario/more/mario.py
# Import modules and functions
from cs50 import get_int

# Promp user for positive 0-23 int
while True:
    height = get_int("Height: ")
    if height in range(24):
        break

row = 0  # Initialize row
for i in range(height):
    # Print left pyramid
    print(" " * (height - row - 1), end="")  # Print spaces
    print("#" * (row + 1), end="")  # Print bricks
    # Print gap
    print(" " * 2, end="")  # 2 spaces
    # Print right pyramid
    print("#" * (row + 1), end="")
    # Print i many rows
    print()
    row += 1  # Increment row